b2sdk/transfer/inbound/file_metadata.py

```diff
@@ -59,3 +59,9 @@ def as_info_dict(self):
             'contentSha1': self.content_sha1,
             'fileInfo': self.file_info,
         }
+
+    @classmethod
+    def _parse_content_sha1(cls, content_sha1):
```

Comment:
I think this method should return a tuple like `(sha1, verified)` and the caller might sha1, _ = cls._parse_content_sha1(content_sha1)` if they don't care, but one they a caller might care and then we should not need to rewrite the parser

Reply:
I'm always trying to not have internal code (not part of API) which is not used. It's cleaner, and once we need it we can add it and it's a good chance it won't be needed.

Reply:
my point is that the unverified checksum flag should be added as a field of this class, now that we do understand it. Why would we expose the checksum but not it's verification status when the server provides it?

Reply:
Still don't agree, but I can leave with that :)